.pylintrc

```diff
@@ -0,0 +1,220 @@
+[MASTER]
+
+# Pickle collected data for later comparisons.
+persistent=no
+
+# Set the cache size for astng objects.
+cache-size=500
+
+# Ignore Py3 files
+ignore=get_references_web.py,get_references_web_single_group.py
+
+
+[REPORTS]
+
+# Set the output format.
+# output-format=sorted-text
+
+# Put messages in a separate file for each module / package specified on the
+# command line instead of printing them on stdout. Reports (if any) will be
+# written in a file name "pylint_global.[txt|html]".
+files-output=no
+
+# Tells whether to display a full report or only the messages.
+reports=no
+
+# Disable the report(s) with the given id(s).
+disable-report=R0001,R0002,R0003,R0004,R0101,R0102,R0201,R0202,R0220,R0401,R0402,R0701,R0801,R0901,R0902,R0903,R0904,R0911,R0912,R0913,R0914,R0915,R0921,R0922,R0923
+
+# Error message template (continued on second line)
+msg-template={msg_id}:{line:3} {obj}: {msg} [{symbol}]
+
+
+[MESSAGES CONTROL]
+# List of checkers and warnings to enable.
+enable=indexing-exception,old-raise-syntax
+
+# List of checkers and warnings to disable.
+# TODO(adarob): Remove wrong-import-order,ungrouped-imports after moving to /py internally.
+disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,file-ignored,multiple-imports,c-extension-no-member,trailing-newlines,unsubscriptable-object,misplaced-comparison-constant,no-member,abstract-method,no-else-return,inconsistent-return-statements,invalid-unary-operand-type,no-name-in-module,arguments-differ,consider-using-enumerate,missing-docstring,too-many-nested-blocks,wrong-import-order,ungrouped-imports,unnecessary-pass,useless-object-inheritance,not-context-manager
```

Comment:
Is this a standard list of warnings to disable?  More generally, where did this particular .pylintrc come from?

Reply:
This is based on tensor2tensor, and is meant to set things up as close as possible to google3. I added a few more to the list that are affected due to the fact that we modify the import names when we import. This will no longer be necessary when we move to the /py directory in google3.